### PR TITLE
chore: Add address field in README

### PR DESCRIPTION
**Pull request by podrivo**
Adds the option `address` in the README.
It closes https://github.com/gamedig/node-gamedig/issues/462.

Thank you @Douile for finding this and giving a nice description! 🙌

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,11 +29,12 @@
   * Minimum Supported Deno Version: `1.39.2`.
   * `deno run --allow-net bin/gamedig.js --type tf2 127.0.0.1`
 * Added code examples.
-* New stable field: `queryPort` - this number indicates what was the port that the query was done on, 0 indicates none if not applicable.
 * Fixed `numplayers` not having a default value.
+* New stable field: `queryPort` - this number indicates what was the port that the query was done on, 0 indicates none if not applicable.
```
Comment (CosminPerRam) [inline CHANGELOG.md:33]: Why this got moved?

**Review by CosminPerRam — APPROVED** [review]
LGTM, thanks, just a ini-tiny question (see above).

File: CHANGELOG.md (hunk 1 above)
Comment (podrivo) [reply CHANGELOG.md:33]: Oops... I guess I was just grouping the options together haha.

Comment (podrivo) [reply CHANGELOG.md]: Fixed! :D

**Pull request merged**